pom.xml

```diff
@@ -182,7 +183,13 @@ THE SOFTWARE.
       <dependency>
         <groupId>com.google.guava</groupId>
         <artifactId>guava</artifactId>
-        <version>11.0.1</version>
+        <version>${guavaVersion}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava-testlib</artifactId>
+        <version>${guavaVersion}</version>
+        <scope>test</scope>
```

Comment:
sorry didn't notice this before but adding scope to a dependency in a dependency Management section is IMHO just evil (at some point you may want to actually depend on this thing and then all the transitive dependencies break - its also not clear in the using pom that it is using it at the right scope - and I have been bitten by this on one too many occasions now).

Other than that LGTM.